prql-compiler/tests/docs/main.rs

```diff
@@ -0,0 +1,75 @@
+use prql_compiler::Options;
```

Comment:
FWIW Rust unfortunately has poor performance with making lots of test binaries. So it would be faster if we could have this as a file within this test https://github.com/PRQL/prql/blob/65706a115a84997c608eaeda38b1aef1240fcec3/web/book/tests/snapshot.rs (which would change to a module, like this file is)

See the link at https://github.com/PRQL/prql/blob/f4eb8c73918b139aa2e49d1c4e82e1652cf956f4/prql-compiler/tests/integration/README.md#L54-L55

(but we can merge this and make the change / I'll make the change, I don't want to slow you down!)